app/routes/overview/components/PublicFrontpage.tsx

```diff
@@ -1,256 +1,187 @@
-              </li>
-            </ul>
-          </div>
-          <div className={styles.facebook}>
-            <h2 className="u-mb">Vår Facebook side</h2>
-            <div className={styles.facebookIframeContainer}>
-              <iframe
-                src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2FAbakusNTNU%2F&amp;tabs=timeline&amp;small_header=true&amp;adapt_container_width=true&amp;hide_cover=false&amp;show_facepile=true&amp;appId=1717809791769695"
-                className={styles.facebookIframe}
-                title="facebook"
-                scrolling="no"
-                frameBorder="0"
-              />
-            </div>
-          </div>
-        </Container>
-      </Container>
-    );
-  }
-}
+      </li>
+      <li>
+        <a href="https://www.ntnu.no/studier/mtdt" target="blank">
+          <i className="fa fa-caret-right" /> Datateknologi
+        </a>
+        <div className={styles.linkDescription}>
+          Datateknologi er en helt sentral del av alle fremtidsrettede
+          teknologier, som for eksempel kunstig intelligens, medisinsk teknologi
+          og søkemotorteknologi.
+        </div>
+      </li>
+      <li>
+        <a href="http://www.ntnu.no/studier/mtkom" target="blank">
+          <i className="fa fa-caret-right" /> Kommunikasjonsteknologi og digital
+          sikkerhet
+        </a>
+        <div className={styles.linkDescription}>
+          Vi bruker stadig mer av livene våre på nett, på jobb som i fritid.
+          Kommunikasjonsteknologi og digital sikkerhet blir stadig viktigere i
+          en digital verden.
+        </div>
+      </li>
+      <li>
+        <Link to="/pages/bedrifter/for-bedrifter">
+          <i className="fa fa-caret-right" /> For bedrifter
+        </Link>
+        <div className={styles.linkDescription}>
+          Her finner du som bedriftsrepresentant informasjon om Abakus{"' "}
+          prosedyrer for bedriftspresentasjoner og andre nyttige fakta.
+        </div>
+      </li>
+      <li>
+        <a href="https://readme.abakus.no">
+          <i className="fa fa-caret-right" /> {readmeIfy('readme')}
+        </a>
+        <div className={styles.linkDescription}>
+          Abakus har sitt eget magasin skrevet av {readmeIfy('readme')}. Her kan
+          du lese om hva abakus driver med og få et innblikk i abakus som
+          organisasjon.
+        </div>
+      </li>
+    </ul>
+  </>
+);
+
+const Facebook = () => (
+  <>
+    <h2 className="u-mb">Vår Facebook side</h2>
+    <div className={styles.facebookIframeContainer}>
+      <iframe
+        src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2FAbakusNTNU%2F&amp;small_header=false&amp;adapt_container_width=true&amp;hide_cover=false&amp;show_facepile=true&amp;appId=1717809791769695"
+        className={styles.facebookIframe}
+        title="facebook"
+        scrolling="no"
+        frameBorder="0"
+      />
+    </div>
+  </>
+);
```

Comment:
In my opinion, we could probably just remove this.

Reply:
I think this might just be due to adblocker or similar, @eikhr ?
Disabling enhanced tracking protection in firefox makes it load (means also that by default the embed is not loaded at all):
![Skjermbilde 2023-04-15 kl  11 54 04](https://user-images.githubusercontent.com/64247965/232206903-e7b362b2-7cf7-4662-a47d-b45294c7eaa5.png)

Also, with 7-8 posts/year, I agree that we might want to drop this.

Reply:
It's also very ugly 😬

Reply:
Yea, maybe it's best to just drop it and give more space to the useful links